packages/squiggle-lang/src/value/index.ts

```diff
@@ -49,7 +50,12 @@ class VArray implements Indexable {
 
   get(key: Value) {
     if (key.type === "Number") {
-      const index = key.value | 0; // TODO - fail on non-integer indices?
+      if (!isInteger(key.value)) {
```

Comment:
I wanted to recommend `Number.isInteger` in place of `lodash` here, as I often do, but I benchmarked both versions and the performance is the same, so it doesn't matter.